src/dbnode/persist/fs/retriever.go

```diff
@@ -491,6 +530,74 @@ func (r *blockRetriever) Stream(
 	return req.toBlock(), nil
 }
 
+func (r *blockRetriever) StreamIndexChecksum(
+	ctx context.Context,
+	shard uint32,
+	id ident.ID,
+	useID bool,
```

Comment:
As before, the caller can likely hold onto the ID here so can we remove `useID`?